modules/pmg_qt/pymol_qt_gui.py

```diff
@@ -692,8 +695,8 @@ def update_pixels(*args):
             width = form.input_width_units.value()
             height = form.input_height_units.value()
             factor = get_factor()
-            form.input_width.setValue(int(width / factor))
-            form.input_height.setValue(int(height / factor))
+            form.input_width.setValue(width / factor)
+            form.input_height.setValue(height / factor)
```

Comment:
Reason for revert? Seems to undo the fix for d6cb36410bb18727bc4fa1918608d9d6d354c62b